DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/RepositoryController.cs

```diff
@@ -70,6 +70,7 @@ public class RepositoryController : BaseApiController
         [Authorize]
         [Route("api/repos/current")]
         [HttpPatch]
+        //TODO: use DTO
         public async Task<IHttpActionResult> UpdateProject(Repository repo)
```

Comment:
Please then ask your colleague to refactor this method and provide me wtih a sequence diagram.